akka-stream-typed/src/test/scala/akka/stream/typed/scaladsl/PubSubSpec.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+package akka.stream.typed.scaladsl
+
+import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import akka.actor.typed.internal.pubsub.TopicImpl
+import akka.actor.typed.pubsub.Topic
+import akka.stream.OverflowStrategy
+import akka.stream.scaladsl.Source
+import akka.stream.testkit.scaladsl.TestSink
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class PubSubSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike {
+
+  implicit def sys = testKit.system.classicSystem
```

Comment:
can't it use the Typed ActorSystem?

Reply:
I didn't think so but turns out there is an `TestSink.apply()` which takes `ClassicActorSystemProvider`. I'll update.